UI/GameInfoCache.cpp

```diff
@@ -405,6 +405,10 @@ class GameInfoWorkItem : public PrioritizedWorkQueueItem {
 					std::lock_guard<std::mutex> lock(info_->lock);
 					info_->paramSFO.ReadSFO(sfoData);
 					info_->ParseParamSFO();
+					if (info_->id.empty() || info_->id == "UCJS10041") {
+						info_->id = g_paramSFO.generateFakeID(gamePath_);
```

Comment:
Not sure, I just mean that if we're changing `id`, maybe we should set `id_version` to match.  Even if it's not used now, it might be in the future.  That's why we have this code for ELFs:

```c++
				std::lock_guard<std::mutex> lock(info_->lock);
				info_->id = "ELF000000";
				info_->id_version = "ELF000000_1.00";
				info_->paramSFOLoaded = true;
```

-[Unknown]